common/rc

```diff
@@ -333,3 +333,19 @@ unload_module() {
 	done
 	return 1
 }
+
+# Return the dmesg log since the start of this test.
+_dmesg_since_test_start()
+{
+	# make shellcheck happy
```

Comment:
This shellcheck exception is a bit ugly. So I suggest to move this _dmesg_since_test_start function from 'common/rc' to 'check. Probably, just before _check_dmesg() function. On top of that, could you replace the 'dmesg' and 'grep -A 9999' call in _check_dmesg with _dmesg_since_test_start? It will avoid the duplication and future improvement easier.

```

@@ -137,7 +143,7 @@ _check_dmesg() {
                return 0
        fi
 
-       dmesg | bash -c "$DMESG_FILTER" | grep -A 9999 "$dmesg_marker" >"${seqres}.dmesg"
+       _dmesg_since_test_start | bash -c "$DMESG_FILTER" >"${seqres}.dmesg"
        grep -q -e "kernel BUG at" \
             -e "WARNING:" \
             -e "BUG:" \


```

Reply:
sure. i will update it soon